src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.FileDao;
+import core.basesyntax.dao.impl.FileDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.CheckDataService;
+import core.basesyntax.service.FruitShopService;
+import core.basesyntax.service.ParseService;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.impl.CheckDataServiceImpl;
+import core.basesyntax.service.impl.FruitShopServiceImpl;
+import core.basesyntax.service.impl.OperationStrategyImpl;
+import core.basesyntax.service.impl.ParseServiceImpl;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import core.basesyntax.service.strategy.BalanceOperationHandler;
+import core.basesyntax.service.strategy.OperationHandler;
+import core.basesyntax.service.strategy.PurchaseOperationHandler;
+import core.basesyntax.service.strategy.ReturnOperationHandler;
+import core.basesyntax.service.strategy.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static final String DAY_ACTIVITY_CSV
+            = "src/main/resources/dayActivities.csv";
+    public static final String DAY_REPORT_CSV
+            = "src/main/resources/dayReport.csv";
+    private static final Map<Operation, OperationHandler> operationHandlerMap = new HashMap<>();
+
+    static {
+        operationHandlerMap.put(Operation.BALANCE, new BalanceOperationHandler());
+        operationHandlerMap.put(Operation.SUPPLY, new SupplyOperationHandler());
+        operationHandlerMap.put(Operation.PURCHASE, new PurchaseOperationHandler());
+        operationHandlerMap.put(Operation.RETURN, new ReturnOperationHandler());
+    }
+
+    public static void main(String[] args) {
+        FileDao fileDao = new FileDaoImpl();

```

Comment:
https://ru.wikipedia.org/wiki/Data_Access_Object
https://www.baeldung.com/java-dao-pattern
See what is 'DAO' and remove it from this class name.  

![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/0b628393-8a6c-446b-ab2b-8c845c171707)